README.md

```diff
@@ -127,12 +127,28 @@ The default executor is responsible for deploying the HelmRelease object passed
 
 Source code for the HelmRelease executor is available [here](https://github.com/Azure/helmrelease-workflow-executor)
 
-### Keptn Executor (Work in progress)
+### Keptn Executor
 
 The Keptn executor is an evaluation executor responsible for running tests on the deployed helm release using the Keptn API and Keptn evaluations engine. The Keptn executor is a custom executor that is chained to the default HelmRelease executor. This allows each release to be evaluated against a set of SLOs/SLIs before it is deployed/updated.
 
 Source code for the Keptn executor is available [here](https://github.com/Azure/keptn-workflow-executor)
 
+#### Argo workflow dashboard
```

Comment:
Since the README is starting to get pretty long, I think it would be useful to have TOC at the top of the README

Reply:
Sure I can add that